xonsh/aliases.py

```diff
@@ -109,6 +114,22 @@ def source_bash(args, stdin=None):
     return source_foreign(args, stdin=stdin)
 
 
+def source_cmd(args, stdin=None):
+    """Simple cmd.exe-specific wrapper around source-foreign."""
+    args = list(args)
+    fpath = locate_binary(args[0])
+    args[0] = fpath if fpath else args[0]
+    args.insert(0, 'cmd')
+    args.append('--interactive=0')
+    args.append('--sourcer=call')
+    args.append('--funcscmd=echo.')
+    args.append('--aliascmd=echo.')
+    args.append('--envcmd=set')
+    args.append('--safe=0')
+
+    return source_foreign(args, stdin=stdin)
```

Comment:
The more I have thought about these wrapper aliases, I think that they can be defined as a simple list of strings rather than these functions: 

```python
DEFAULT_ALIASES ['source-cmd'] = ['source-foriegn', 'cmd', '--interactive=0', '--sourcer=call',  
        '--funcscmd=echo.', '--aliascmd=echo.', '--envcmd=set', '--safe=0']
```